### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by laritakr**
# Story

Puts views into right position to get picked up.
Loads CSS.
Fixes assets precompile.
Sets up appearance colors and fonts with appropriate defaults. 
Actions 

Refs
- https://github.com/scientist-softserv/adventist-dl/issues/562

# Expected Behavior Before Changes

Layout is Hyrax basic.

# Expected Behavior After Changes

Views mostly match current adventist.

# Screenshots / Video

<details>
<summary></summary>

**Style matches staging:**
Screenshot 2023-10-02 at 10 24 25 PM

**Fonts can be managed from appearance tab:**
Screenshot 2023-10-02 at 11 19 32 PM

</details>

# Notes

Prior custom CSS overrides a number of the appearance colors. This was not changed.

### CODE REVIEW

File: app/assets/stylesheets/adl-overrides/adl-overrides.scss [hunk 1]
```diff
@@ -1,7 +1,6 @@
 @import url("https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap");
 
 body.public-facing {
-  font-family: "Source Sans Pro" !important;
```
Comment (laritakr) [inline app/assets/stylesheets/adl-overrides/adl-overrides.scss:4]: Removing font so the font selection in the appearance tab will work.

File: app/assets/stylesheets/adl-overrides/viewer.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.viewer-wrapper {
```
Comment (laritakr) [inline app/assets/stylesheets/adl-overrides/viewer.scss:1]: Added due to custom css in the adventist_dl repo that was different than hyku.

**Comment by ShanaLMoore** [comment]
Could you add an after screenshot to this PR please?

**Comment by ShanaLMoore** [comment]
Sorry I missed clicking on the arrow. I see the drop down screenshot now. @laritakr Would a user have to switch to this theme?

**Review by ShanaLMoore — APPROVED** [review]

**Comment by laritakr** [comment]
> Sorry I missed clicking on the arrow. I see the drop down screenshot now. @laritakr Would a user have to switch to this theme?

No themes at this point... they have a lot of custom CSS and that plus the complexity of how theming works with routes (and trying to fit that into the knapsack world) made doing it as a theme out of scope at this point. The goal for now is to reproduce what they already had, though I had hoped a theme would be a manageable way to do it.

**Pull request merged**